Source/Scene/PostProcessorScene.js

```diff
@@ -20,23 +20,247 @@ define([
      */
     function PostProcessorScene() {
         this.blackAndWhiteStage = createBlackAndWhiteStage();
+        this.blackAndWhiteStage.show = false;
         this.brightnessStage = createBrightnessStage();
+        this.brightnessStage.show = false;
         this.eightBitStage = createEightBitStage();
+        this.eightBitStage.show = false;
         this.compositeTextureStage = createCompositeTextureStage();
         this.compositeTextureStage.show = false;
+        this.LensFlareStage = createLensFlareStage();
 
         var stages = [
             this.blackAndWhiteStage,
             this.brightnessStage,
             this.eightBitStage,
-            this.compositeTextureStage
+            this.compositeTextureStage,
+            this.LensFlareStage
         ];
 
         this._postProcessor = new PostProcessor({
             stages : stages
         });
     }
 
+    function createLensFlareStage() {
+        var url_Dirt = buildModuleUrl('Assets/Textures/LensFlare/DirtMask.jpg');
+        var url_Star = buildModuleUrl('Assets/Textures/LensFlare/StarBurst.jpg');
+
+        var uniformValues = {
+
+            DirtTexture: url_Dirt,
+            StarTexture: url_Star,
+
+            Intensity: 2.0,
+
+            Distortion: 5.0,
+            NumofGhosts: 4,
+            GhostDispersal: 0.4,
+
+            HaloWidth: 0.4,
+
+            EarthRadius: 0.0
+        };
+
+        var LensFlareShader =
+            'uniform sampler2D u_colorTexture; \n' +
+            'uniform sampler2D u_DirtTexture; \n' +
+            'uniform sampler2D u_StarTexture; \n' +
+
+            'uniform float u_Distortion; \n' +
```

Comment:
If it is currently possible with the post-processing framework, I would combine all these floats into a `vec4`...and generally pack uniforms together as much as possible.  We'll still want to expose these in JavaScript as separate variables then map to the uniforms x, y, z, w components like we do [here](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Shaders/BillboardCollectionVS.glsl).

Since this shader isn't called often, this isn't a huge performance issue, but this is the common practice throughout Cesium.

If the post-process framework doesn't support this, @lilleyse can add it and we can wait on this change or @byumjin you are welcome to go for it.